utilities/BuildUtilities.groovy

```diff
@@ -789,4 +789,26 @@ def getShortGitHash(String buildFile) {
 	if (abbrevGitHash != null ) return abbrevGitHash
 	if (props.verbose) println "*! Could not obtain abbreviated githash for buildFile $buildFile"
 	return null
-}
\ No newline at end of file
+}
+
+/**
+ * print the logicalFile attributes and indicates if attributes are overriden
+ * by file property overrides.
+ * 
+ * //File attributes: CICS=Yes, SQL=Yes*, DLI=No, MQ=No
```

Comment:
Update this comment to match the println.
Also, for readability, perhaps add a comment that explains what the "*" means.